dll/win32/shell32/shelldesktop/CDesktopBrowser.cpp

```diff
@@ -471,7 +471,7 @@ LRESULT CDesktopBrowser::OnShowOptionsDlg(UINT uMsg, WPARAM wParam, LPARAM lPara
         case 2:
         case 7:
 #endif
-            ShowFolderOptionsDialog(PtrToUlong(wParam), TRUE);
+            ShowFolderOptionsDialog((UINT)(UINT_PTR)wParam, TRUE);
```

Comment:
You should be able to remove the extra `(UINT_PTR)`, since WPARAM (and LPARAM) are _already_ UINT_PTRs.

Reply:
cc @tkreuzer who is usually the guy who adds calls like the PtrToULong() for improving x64-compat. His words can typically be trusted for any changes in that aspect. Not sure whether this one here originated from him.